README.md

```diff
@@ -4,8 +4,41 @@ Binaries for Windows, Ubuntu and MacOSX are available on the Releases tab:
 
 https://github.com/obsidianplatform/Obsidian-Qt/releases
 
-To build for Linux, see build.sh and src/buildquick.sh.
-For a build with static linking, look at new-build.sh.
+
+## Building Obsidian-QT
+### Ubuntu
+To install dependencies and build Obsidian-QT under Ubuntu,
+open a terminal and then do the following:
+
+
+clone the github repository
+```
+git clone https://github.com/obsidianplatform/Obsidian-Qt
+```
+move into Obsidian-Qt folder
+```
+cd Obsidian-Qt
+```
+run buildscript to build binarys,
+when asked for a RPC-User and a RPC-Password,
+choose a username and a secure password,
+enter them and press **Enter**
+
+```
+./new-build.sh
+```
+This generates a folder called *binarys* where the newly build binarys for
+*Obsidian-Qt* and *obsidiand* can be found.
+
+#### statically linked builds
+Statically linked builds are prefered when the binary should
+be runnable on different other GNU/Linux systems then Ubuntu.
```

Comment:
Should clean up this line to just say:
be executable on other GNU/Linux systems.